src/token/types.ts

```diff
@@ -1 +1,19 @@
 // 🌲☀️🌲
+
+import { Address } from 'viem';
+
+export type Token = {
+  name: string;
+  currencyCode: string;
+  imageURL: string;
+  blockchain: string;
+  address: Address;
+  chainId: number;
+  decimals: number;
+  uuid: string;
+};
+
+export type TokenChipReact = {
+  token: Token;
+  onClick: (token: Token) => void;
```

Comment:
@kyhyco open question, do we really think `onClick` is need it here? After all someone could just add an `onClick` from the outer component.